tools/ovc/openvino/tools/ovc/moc_frontend/offline_transformations.py

```diff
@@ -89,7 +89,7 @@ def apply_user_transformations(func: object, transforms: list):
 
 def apply_moc_transformations(func: object):
     from openvino._offline_transformations import apply_moc_transformations  # pylint: disable=import-error,no-name-in-module
-    apply_moc_transformations(func, cf=False, smart_reshape=True)
+    apply_moc_transformations(func, cf=True, smart_reshape=False)
```

Comment:
do we need this wrapper for `apply_moc_transformations`? Why can't we directly call it?

Reply:
No, we don't need it in OVC. Removed it.